Adding eplotly recipe

GioBo opened the pull request:
### Brief summary of what the package does

This package allows to create Plotly charts directly from Emacs.

### Direct link to the package repository

https://codeberg.org/GioBo/eplotly

### Your association with the package

I am the creator of the package

### Relevant communications with the upstream package maintainer

 **None needed**

### Checklist

<!-- Please confirm by replacing `[]` with `[x]`: -->

- [x] The package is released under a [GPL-Compatible Free Software License](https://www.gnu.org/licenses/license-list.en.html#GPLCompatibleLicenses)
- [x] I've read [CONTRIBUTING.org](https://github.com/melpa/melpa/blob/master/CONTRIBUTING.org)
- [x] I've used the latest version of [package-lint](https://github.com/purcell/package-lint) to check for packaging issues, and addressed its feedback
- [x] My elisp byte-compiles cleanly
- [x] I've used `M-x checkdoc` to check the package's documentation strings
- [x] I've built and installed the package using the instructions in [CONTRIBUTING.org](https://github.com/melpa/melpa/blob/master/CONTRIBUTING.org)

<!-- After submitting, please fix any problems the CI reports. -->

riscy commented:
Thanks for the update.  Here's the latest -

⸺ `eplotly.el` with byte-compile using Emacs 30.1:
```
eplotly.el:644:2: Warning: docstring has wrong usage of unescaped single quotes (use \=' or different quoting such as `...')
```
This is a legitimate warning in this case because the builtin help will subtly modify your example code, leading to it being invalid elisp code, e.g.
```elisp
(let*
  ((ro
`(
     (eplotly-fun
      (
	     (parabola -10 10 :color "red" :name "parabola" :dash "solid")
	     (,(tangent-at #’parabola #’parabola-deriv 3) -10 10
```
^ notice the use of curly quotes instead of apostrophes in how the help is rendered.

On closer inspection I'm also noticing you're using a sort of C-style for your function signatures.  In elisp there should be a space between the function name and argument list in the following places:

- eplotly.el#L77: No space between the function name and argument list
- eplotly.el#L101: No space between the function name and argument list
- eplotly.el#L106: No space between the function name and argument list
- eplotly.el#L114: No space between the function name and argument list
- eplotly.el#L131: No space between the function name and argument list
- eplotly.el#L166: No space between the function name and argument list
- eplotly.el#L772: No space between the function name and argument list
- eplotly.el#L778: No space between the function name and argument list
- eplotly.el#L190: No space between the function name and argument list
- eplotly.el#L267: No space between the function name and argument list
- eplotly.el#L298: No space between the function name and argument list
- eplotly.el#L323: No space between the function name and argument list
- eplotly.el#L345: No space between the function name and argument list
- eplotly.el#L360: No space between the function name and argument list
- eplotly.el#L383: No space between the function name and argument list
- eplotly.el#L398: No space between the function name and argument list
- eplotly.el#L425: No space between the function name and argument list
- eplotly.el#L441: No space between the function name and argument list
- eplotly.el#L471: No space between the function name and argument list
- eplotly.el#L487: No space between the function name and argument list
- eplotly.el#L517: No space between the function name and argument list
- eplotly.el#L533: No space between the function name and argument list
- eplotly.el#L577: No space between the function name and argument list
- eplotly.el#L593: No space between the function name and argument list
- eplotly.el#L627: No space between the function name and argument list
- eplotly.el#L644: No space between the function name and argument list
- eplotly.el#L712: No space between the function name and argument list
- eplotly.el#L801: No space between the function name and argument list
- eplotly.el#L843: No space between the function name and argument list

GioBo commented:
Thanks a lot for the additional feedback; I've upgraded to Emacs 30.1 and could reproduce the warning (which I couldn't see in Emacs 28). I've fixed that and also added a space between every function name and its argument list.
Thanks.

riscy commented:
Thank you!  I think we're all set.

Not a blocker, but fyi your `;;; Commentary` in the MELPA index will render like this:

```elisp
Create Plotly charts

;; Installation

;;; MELPA

If you installed from MELPA, you're done.

;;; Manual

Install these required packages:

+ jack
+ f

Then put this file in your load-path, and put this in your init
file:

(require 'eplotly)

;; Usage

Run:

M-x eplotly-mode

;; Tips
```
I think it's best if a package's commentary starts with an explanation of what it's for (what's eplotly?), rather than how to install it (which most emacs users already know or will know soon).  Also noting the empty "Tips" section at the bottom.

I'll merge this now though - thank you again!
